docs/kratos/manage-identities/30_account-recovery.mdx

```diff
@@ -1,25 +1,27 @@
 ---
 id: account-recovery
-title: Learn about the account recovery flow initiated by admins
+title: Initiate account recovery for a user
 sidebar_label: Administrative account recovery
 ---
 
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import CodeBlock from "@theme/CodeBlock"
+import SelfHostedKratos from "@site/src/components/SelfHostedKratos"
+
 You can initiate account recovery for identities using the admin API endpoints.
 
+For more detailed information and general guidelines on account recovery, take a look at the
+[account recovery and password reset](../self-service/flows/account-recovery-password-reset.mdx) section.
```

Comment:
"account recovery" w/o reset as mentioned in other comments?

Reply:
First we need to decide, what to name that page!